diff --git a/ckanext/issues/tests/logic/action/test_issue.py b/ckanext/issues/tests/logic/action/test_issue.py
index 76c7474..7623a5e 100644
--- a/ckanext/issues/tests/logic/action/test_issue.py
+++ b/ckanext/issues/tests/logic/action/test_issue.py
@@ -281,6 +281,7 @@ class TestIssueComment(object):
         closed = helpers.call_action(
             'issue_update',
             issue_number=issue['number'],
+            description='some description',
             dataset_id=dataset['id'],
             context={'user': user['name']},
             status='closed'
@@ -513,6 +514,7 @@ class TestIssueUpdate(object):
             'issue_update',
             context={'user': user['name']},
             issue_number=issue['number'],
+            description='new description',
             dataset_id=dataset['id'],
             title='new title',
             description='new description'
