diff --git a/ckanext/issues/tests/logic/action/test_issue.py b/ckanext/issues/tests/logic/action/test_issue.py
index 76c7474..e301bed 100644
--- a/ckanext/issues/tests/logic/action/test_issue.py
+++ b/ckanext/issues/tests/logic/action/test_issue.py
@@ -281,6 +281,7 @@ class TestIssueComment(object):
         closed = helpers.call_action(
             'issue_update',
             issue_number=issue['number'],
+            description='some description',
             dataset_id=dataset['id'],
             context={'user': user['name']},
             status='closed'
@@ -539,6 +540,7 @@ class TestIssueUpdate(object):
             dataset_id=dataset['id'],
             issue_number=issue['number'],
             status='closed'
+            description='some description',
         )
         assert 'closed' == closed['status']
 
