diff --git a/ckan/lib/datapreview.py b/ckan/lib/datapreview.py
index 9dbd9e562..10d478d64 100644
--- a/ckan/lib/datapreview.py
+++ b/ckan/lib/datapreview.py
@@ -124,7 +124,12 @@ def get_default_view_plugins(
 
     Returns a list of IResourceView plugins
     '''
-    default_view_types = config.get('ckan.views.default_views')
+    default_view_types = config.get('ckan.views.default_views', [])
+
+    if isinstance(default_view_types, str):
+        default_view_types = default_view_types.split()
+    elif not isinstance(default_view_types, list):
+        default_view_types = []
 
     default_view_plugins = []
     for view_type in default_view_types:
