diff --git a/ckan/model/__init__.py b/ckan/model/__init__.py
index 595ae3876..a03e50c25 100644
--- a/ckan/model/__init__.py
+++ b/ckan/model/__init__.py
@@ -237,6 +237,11 @@ class Repository():
             warnings.filterwarnings('ignore', '.*(reflection|tsvector).*')
             meta.metadata.reflect(engine)
 
+        tables = meta.metadata.sorted_tables
+        for table in tables:
+            if table.name == "spatial_ref_sys":
+                meta.metadata.remove(table)
+
         with engine.begin() as conn:
             meta.metadata.drop_all(conn)
 
