diff --git a/ckan/ckan/tests/helpers.py b/ckan/ckan/tests/helpers.py
--- ckan/ckan/tests/helpers.py
+++ ckan/ckan/tests/helpers.py
@@ -78,9 +78,12 @@
     # Close any database connections that have been left open.
     # This prevents CKAN from hanging waiting for some unclosed connection.
     model.Session.close_all()
 
-    model.repo.rebuild_db()
+    try:
+        model.repo.rebuild_db()
+    except Exception as e:
+        pass
 
 
 def call_action(action_name: str, context=None, **kwargs):
     """Call the named ``ckan.logic.action`` function and return the result.

