diff --git a/ckanext/harvest/logic/action/update.py b/ckanext/harvest/logic/action/update.py
index 700e397..52a1685 100644
--- a/ckanext/harvest/logic/action/update.py
+++ b/ckanext/harvest/logic/action/update.py
@@ -125,8 +125,9 @@ def harvest_source_clear(context, data_dict):
             WHERE harvest_source_id = '{harvest_source_id}');"""
 
     sql += """
-        CREATE TEMP TABLE package_ids_to_delete AS (
-            SELECT id FROM package WHERE state = 'to_delete');
+        DROP TABLE IF EXISTS package_ids_to_delete;
+        CREATE TEMP TABLE package_ids_to_delete ON COMMIT DROP AS (
+             SELECT id FROM package WHERE state = 'to_delete');
         """
 
     sql += f"""
